src/components/GameView/GameMenu.vue

```diff
@@ -14,10 +14,15 @@
           <rules-dialog :hideActivator="true" :show="showRulesDialog" @close="showRulesDialog = false" />
         </v-list-item>
         <!-- Concede Dialog (Initiate + Confirm) -->
-        <v-list-item data-cy="concede-initiate" @click.stop="openConcedeDialog">Concede</v-list-item>
-        <v-list-item data-cy="stalemate-initiate" @click.stop="openStalemateDialog">
-          Request Stalemate
-        </v-list-item>
+        <v-item-group v-if="!isSpectating">
+          <v-list-item data-cy="concede-initiate" @click.stop="openConcedeDialog"> Concede</v-list-item>
+          <v-list-item data-cy="stalemate-initiate" @click.stop="openStalemateDialog">
+            Request Stalemate
+          </v-list-item>
+        </v-item-group>
+
+        <!-- Stop Spectating -->
+        <v-list-item v-else data-cy="stop-spectating" @click.stop="stopSpectate"> Go Home</v-list-item>
```

Comment:
It's often easier to read if/else logic if you can avoid the need for !'s. So I'd prefer to put the spectating case above the non-specating stuff. Additionally we don't want a v-item-group because we don't want to create a functional difference in the UI, we just want to group these tags together for use with the v-if directive. So we should use a `<template>` as the wrapper instead (template is the generic wrapper in vue that doesn't get actually rendered -- it's the correct way to group things together to apply directives to the group)

```suggestion
        <!-- Stop Spectating -->
        <v-list-item v-if="isSpectating" data-cy="stop-spectating" @click.stop="stopSpectate"> Go Home</v-list-item>
        <template v-else="!isSpectating">
          <v-list-item data-cy="concede-initiate" @click.stop="openConcedeDialog"> Concede</v-list-item>
          <v-list-item data-cy="stalemate-initiate" @click.stop="openStalemateDialog">
            Request Stalemate
          </v-list-item>
        </template>
```

Reply:
***v-list-group